### DISCUSSION TITLE
第18期：马斯克都开始玩悟空了 | 信息差——独立开发者出海周刊

### DISCUSSION THREAD

**Discussion by giscus** [weijunext/gapis.money#29] [General]
# 第18期：马斯克都开始玩悟空了 | 信息差——独立开发者出海周刊

Knowledge is power, info-gap is money!「信息差——独立开发者出海周刊」是一个帮助独立开发者缩小信息差的技术周刊。

https://gapis.money/weekly/2024-09-09_018

**Comment by kun321** [comment]
不只是悟空的脸, 比如金池长老, 虎先锋, 甚至,石双双?😄
